.github/rm-invalid-lowest-lock-files.php

```diff
@@ -104,8 +104,7 @@ function getContentHash(array $composerJson)
     }
 }
 
-if (!$referencedCommits || (isset($_SERVER['TRAVIS_PULL_REQUEST']) && 'false' !== $_SERVER['TRAVIS_PULL_REQUEST'])) {
-    // cached commits cannot be stale for PRs
```

Comment:
it seems they can: https://travis-ci.org/symfony/symfony/jobs/438809475

Reply:
That's good: now than we use flex as an accelerator, this is not really slow anymore.

Reply:
thanks